components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/util/FrameworkConstants.java

```diff
@@ -214,6 +214,9 @@ public abstract class FrameworkConstants {
     public static final String NONCE_COOKIE_WHITELISTED_AUTHENTICATORS_CONFIG =
             "SessionNonceCookie.WhitelistedAuthenticator";
 
+    public static final String BLOCKED_USERSTORE_DOMAINS_LIST = "BlockedUserStoreDomains";
+    public static final String BLOCKED_USERSTORE_DOMAINS_SEPARATOR = ",";
```

Comment:
what is the use case of this BlockedUserStoreDomains? Can you please add a sample value in the description of the PR and give a small description in the code as well

Reply:
BlockedUserStoreDomains can be configured for authenticators. When users are retrieved, users residing in the blocked user stores will be omitted. In this specific case, it is added to prevent the same user from two user stores being duplicated.

eg: 
```
[authentication.authenticator.email_otp]
parameters.BlockedUserStoreDomains = "CUSTOMER"
```